Say Their Names/Source/Networking/NetworkSessionMock.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Say Their Names
+//
+//  Created by evilpenguin.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - NetworkSessionMock
+typealias NetworkSessionMock<T> = NetworkSession<T>
+extension NetworkSessionMock {
+    public class func loadMock<T>(_ download: NetworkTask<T>, completion: @escaping (T?, TimeInterval, NetworkError?) -> Swift.Void) {
+//        guard let modelClass = T.self as? BaseModel.Type else { return }
+//
+//        let time = Date()
+//        let (result, networkError) = download.parse(modelClass.mockJson(), nil, nil)
+//        completion(result, Date().timeIntervalSince(time), networkError)
+    }
+}
```

Comment:
Is this for testing purposes? If so, does it make sense to have this in the `Say Their Names Tests/` dir? Also, does it make sense to mock network requests, if that's what we are doing here, with some automated classes?  I've have some luck with https://github.com/uber/mockolo

Reply:
I removed this and we are using Alamofire now. We should def use Mockolo. Great framework.